src/components/DataViewer/DataViewer.tsx

```diff
@@ -86,15 +115,10 @@ const DataViewer: React.FC<DataViewerProps> = (props) => {
     const closeModal = (): void => {
         props.onRequestClose();
     };
-    if (!props.isOpen) {
-        return <></>;
-    }
-
-    Modal.setAppElement("#modal-parent");
```

Comment:
Might want to check https://reactcommunity.org/react-modal/accessibility/ to make sure removing this line is OK. Because I can see that it currently passes the accessibility checks which is great but I don't know it it is necessary to include this line and the div id in SampleDataWithButton to allow screen readers to actually read it.

Reply:
if it is required then can it be encapsulated at a component-level, so callers of DataViewer don't have to 'know' to put in a div with a certain ID in?

also, for reuse, this may violate certain rules like duplicate IDs... 🤔 may be worth investigating if we pass a generated ID via React or something similar!

Reply:
additionally I fully agree with not just deleting things because 'accessibility test still passes' - I might not have set up the tests fully, or this may be 'better' extended accessibility than what's currently being checked!
We should make an informed choice for this sort of thing, rather than just writing code that passes the tests :)

Reply:
our newest iteration will use our general modal component which is from the mui library - hence the id is no longer needed for this new library

Reply:
This still uses React-Modal so please keep this in! We want it to be as accessible as possible at all points - even if it's getting replaced.
(Which will require adding a div with that ID back in!)

Reply:
Ok! However our newest version has already replaced with MUI library, but will bear this in mind!